adafruit_led_animation/animation/rainbowcomet.py

```diff
@@ -60,7 +60,7 @@ def __init__(
         ring=False,
     ):
         if step == 0:
-            self._colorwheel_step = int(256 / tail_length)
+            self._colorwheel_step = max(int(256 / tail_length), 1)
```

Comment:
Not your code but simplifies things by using integer division.

```suggestion
            self._colorwheel_step = max(256 // tail_length, 1)
```

Reply:
Thank you, changed to integer division in the latest commit.